src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -149,19 +171,22 @@ class TaskForm extends React.Component<Props, State> {
 
     handleValidSubmit = (): void => {
         const { id, createTask, editTask, editMode, taskType } = this.props;
-        const { message, approvers, dueDate, isValid } = this.state;
+        const { message, approvers, addedAssignees, removedAssignees, dueDate, isValid } = this.state;
         const dueDateString = dueDate && dueDate.toISOString();
```

Comment:
btw `dueDateString` is not really a string if `dueDate` is not a date object.

Reply:
fair point. I've wondered if we should get more into the habit of typing `const`s like this anyhow...